history-of-the-quakers/original/24-book-2-chapter-15.adoc

```diff
@@ -214,10 +231,17 @@ who kindly invited him to fix his residence with him, and he would make him a wa
 adding his admiration at the Englishmen`'s notion of the divine being,
 who dealt so cruelly with one another about their God.
 
-+++___________+++Pudet hac opprobria nobis, Et dici potuisse, et non potuisse refelli.
+[quote]
+____
+_+++___________+++Pudet hac opprobria nobis, Et dici potuisse, et non potuisse refelli._
+____
 
-That such reproaches stain the christian name, And stain with reason,
-is the christian`'s shame.
+[quote]
+____
+That such reproaches stain the christian name,
+And stain with reason, is the christian`'s shame.
+____
+// I did the quote like this above because it looked funny in the pdf the other ways I tried...if there's another way, that's cool too.
```

Comment:
Because I'm sorting out the footnote stuff in this PR, I'm not going to have you change it, but in case it helps future spots, I ended up changing this to:

```ASCIIDOC
[verse]
____
Pudet hac opprobria nobis,
Et dici potuisse, et non potuisse refelli.

That such reproaches stain the christian name,
And stain with reason, is the christian`'s shame.
____
```